src/test/java/no/nav/tag/sykefravarsstatistikk/api/tapteDagsverkForKostnadsberegning/TapteDagsverkServiceTest.java

```diff
@@ -49,4 +52,59 @@ public void hentTapteDagsverkFraDeSiste4Kvartalene__skal_sende_rikitg_underenhet
 
         verify(repository, times(1)).hentTapteDagsverkFor4Kvartaler(any(), eq(orgnr));
     }
+
+    @Test
+    public void oppsummerTapteDagsverk__skal_sende_riktige_årstall_og_kvartaler() {
+        List<KvartalsvisTapteDagsverk> kvartalsvisTapteDagsverksListe = Arrays.asList(
+                new KvartalsvisTapteDagsverk(new BigDecimal(100), 2019, 3, 50),
+                new KvartalsvisTapteDagsverk(new BigDecimal(100), 2019, 2, 50),
+                new KvartalsvisTapteDagsverk(new BigDecimal(100), 2019, 1, 50),
+                new KvartalsvisTapteDagsverk(new BigDecimal(100), 2018, 4, 50)
+        );
+        List<ÅrstallOgKvartal> årstallOgKvartal = Arrays.asList(
+                new ÅrstallOgKvartal(2019, 3),
+                new ÅrstallOgKvartal(2019, 2),
+                new ÅrstallOgKvartal(2019, 1),
+                new ÅrstallOgKvartal(2018, 4)
+        );
+        tapteDagsverkService.hentOppsummertTapteDagsverk(null);
+
+        verify(repository, times(1)).hentTapteDagsverkFor4Kvartaler(eq(årstallOgKvartal), any());
+        //tapteDagsverkService.oppsummerTapteDagsverk()
+        //TapteDagsverk
+        //tapteDagsverkService.hentTapteDagsverkFraDeSiste4Kvartalene(null);
+
+        //verify(repository, times(1)).hentTapteDagsverkFor4Kvartaler(eq(årstallOgKvartal), any());
+    }
+
+    @Test
+    public void oppsummerTapteDagsverk__skal_sende_rikitg_underenhet() {
+        Orgnr orgnr = etOrgnr();
+        tapteDagsverkService.hentOppsummertTapteDagsverk(orgnr);
+
+        verify(repository, times(1)).hentTapteDagsverkFor4Kvartaler(any(), eq(orgnr));
+    }
+
+    @Test
+    public void oppsummerTapteDagsverk__skal_ikke_maskere_her() {
```

Comment:
Bra med tester 😄 Hva med å forklare litt grundigere hva testen gjør, så man ikke trenger å lese hele testen for å finne det ut? F.eks. "__skal_ikke_maskere_hvis_ingen_av_kvartalene_er_maskert"? og tilsvarende for neste test